go.mod

```diff
@@ -4,18 +4,18 @@ go 1.14
 
 require (
 	github.com/ghodss/yaml v1.0.0
-	github.com/giantswarm/apiextensions/v2 v2.6.2
+	github.com/giantswarm/apiextensions/v3 v3.8.1-0.20201120151140-f03bde4123c1
```

Comment:
why using non released version here? aybe mistake?

Reply:
Sorry forgot to mention - this is latest `apiextensions` master. I will tag it later on but wanted to create some PRs for cluster operator already.